platform/src/main/asciidoc/upgrading.adoc

```diff
@@ -12,6 +12,9 @@ Platform.
 [[upgrading-dependency-management]]
 == Changes to dependency management
 
+| `org.springframework.boot:spring-boot-starter-data-geode`
```

Comment:
I don't think Boot's ever had a `spring-boot-starter-data-geode` module (and looking at Maven Central and repo.spring.io seem to confirm it) so there's no need for this.

Reply:
Right!